lumeer-core/src/main/java/io/lumeer/core/facade/LinkInstanceFacade.java

```diff
@@ -19,27 +19,71 @@
 
 package io.lumeer.core.facade;
 
+import io.lumeer.api.model.Collection;
 import io.lumeer.api.model.LinkInstance;
-import io.lumeer.storage.api.query.SearchQuery;
+import io.lumeer.api.model.LinkType;
+import io.lumeer.api.model.Query;
+import io.lumeer.api.model.Role;
+import io.lumeer.core.PermissionsChecker;
+import io.lumeer.storage.api.dao.CollectionDao;
+import io.lumeer.storage.api.dao.LinkInstanceDao;
+import io.lumeer.storage.api.dao.LinkTypeDao;
 
+import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import javax.inject.Inject;
 
 public class LinkInstanceFacade {
```

Comment:
Scope is not defined here and so `@Dependent` is used by default. We should rather explicitly define scope in order to prevent confusion.